executor/src/main/java/org/k8loud/executor/action/Action.java

```diff
@@ -20,5 +24,23 @@ protected Action(Params params) throws ActionException {
 
     public abstract void unpackParams(Params params) throws ActionException;
 
-    public abstract ExecutionRS perform();
+    public ExecutionRS perform() {
+        String result;
+        try {
+            result = performBody();
```

Comment:
`action.performAction()`, isn't it kind of repetetive? 
I'll change it to `execute` though since we have `ExecutionService`, `ExecutionRQ`, `ExecutionRS`